metaflow/plugins/argo/argo_events_decorator.py

```diff
@@ -0,0 +1,154 @@
+from metaflow.decorators import FlowDecorator
+from metaflow.exception import MetaflowException
+from metaflow.util import is_stringish
+
+# TODO: Support dynamic parameter mapping through a context object that exposes
```

Comment:
As discussed, we should enunciate some of the rules here (the ones we agreed to during the meeting) around parameter mapping.

Reply:
it is listed as a TODO at the moment in the description of this PR.